src/Query.php

```diff
@@ -323,6 +349,26 @@ public function per_page_control( bool $set = true, string $label = 'Limit' ): s
 		return $this;
 	}
 
+	/**
+	 * Adds a limit control to the query. Also adds the query args by default
+	 *
+	 * @param bool $set
+	 * @param string $label
+	 *
+	 * @return Query
+	 */
+	public function multisite_control( bool $set = true, string $label = 'Site' ): static {
+		if ( $set === false ) {
+			$this->config_flags['multisite_control'] = false;
+
+			return $this;
+		} else {
+			$this->config_flags['multisite_control'] = new Multisite_Control( $this->name, $label );
+		}
```

Comment:
Only allow enabling when multisite is enabled. Check with something like is_multisite()